network/vif-route-qubes

```diff
@@ -47,6 +47,14 @@ network_hooks() {
     fi
 }
 
+conntrack_purge () {
+    local n output deleted msg
+    n='(0|[1-9][0-9]*)' msg='flow entries have been deleted\.$'
+    deleted="^conntrack v$n\\.$n\\.$n \\(conntrack-tools\\): $n $msg"
+    output=$(LC_ALL=C exec conntrack -D "$@" 2>&1 >/dev/null) || :
```

Comment:
This requires adding a dependency on `conntrack-tools`, to all present packaging. Otherwise the command is not available, which prevents any `vif*` interface from coming up...

Reply:
Oops!  Fixed, at least for Arch, Debian, and Fedora.